app/assets/stylesheets/pages/components/_register_mobile.scss

```diff
@@ -0,0 +1,46 @@
+.register_page{
+  background-color: #caf0f8;
```

Comment:
What do you think of ordering all your style properties alphabetically?

Reply:
In next issue we are resolving